cypress/e2e/CMS/$InfographicCreation/index.js

```diff
@@ -0,0 +1,376 @@
+});
+
+And('user clicks on {string} button to select media', (selectInfograhic) => {
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_embedded_infographic_browser').find(`input[id='edit-submit'][value='${selectInfograhic}']`).click({ force: true });
+});
+
+And('{string} dropdown has the following options', (labelText, dataTable) => {
+    cy.get(`label[class="js-form-required form-required"]:contains("${labelText}")`).should('be.visible');
+    for (const { options } of dataTable.hashes()) {
+        cy.get(`select[id*="edit-attributes-data-entity-embed-display"] option:contains("${options}")`).should('exist');
+    }
+});
+
+And('user selects a checkbox next to title with url {string} from the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).parent().parent().find('input.form-checkbox').check();
+});
+
+And('{string} dropdown displays {string}', (labelText, displayOption) => {
+    cy.get(`select[id*="edit-attributes-data-entity-embed-display"] option:contains("${displayOption}")`).should('have.attr', 'selected', 'selected');
+});
+
+And('{string} label has the following options', (labelText, dataTable) => {
+    cy.get(`legend span:contains("${labelText}")`).should('be.visible');
+    for (const { options } of dataTable.hashes()) {
+        cy.get(`div.fieldset-wrapper label:contains("${options}")`).should('be.visible');
+    }
+});
+
+And('the radio button {string} is selected by default under {string}', (alignPosition, labelText) => {
+    cy.get(`label:contains("${alignPosition}")`).parent().find(`div[class*="js-form-item-attributes-data-align"] input[checked="checked"]`).should('be.visible');
+});
+
+And('user selects {string} radio button under {string}', (alignPosition, labelText) => {
+    cy.get(`div.fieldset-wrapper input[value='${alignPosition}']`).click({ force: true });
+});
+
+And('user clicks on {string} button to select infographic', (embedButton) => {
+    cy.get(`div.ui-dialog-buttonset.form-actions button:contains("${embedButton}")`).click({ force: true });
+});
+
+And('user select {string} from the {string} dropdown', (selectItem, labelText) => {
+    cy.get(`label[class="js-form-required form-required"]:contains("${labelText}")`).should('be.visible');
+    cy.get(`select[id*="edit-attributes-data-entity-embed-display"]`).select(selectItem);
+});
+
+And('user clicks on {string} to add a body section', (option) => {
+    cy.get(`input[value='${option}']`).click();
+    cy.get("div[data-drupal-selector='edit-field-article-body-1-top-paragraph-type-title']").should('exist');
+});
+
+And('user clicks {string} button for adding banner', (bannerBtn) => {
+    cy.get(`input[id*="edit-field-hero-banner-add-more"]`).click({ force: true });
+});
+
+And('user uploads test file {string} as {string} Version', (fileName, mode) => {
+    if (mode == 'Desktop') {
+        cy.get(`label[id*="subform-field-desktop-hero"]:contains("${mode}")`).should('be.visible');
```

Comment:
hint - label[id*="subform-field-desktop-hero"] already contains desktop, so you should just pass it directly
 label[id*="subform-field-${mode}-hero"]
and make mode lowercase in your feature file

Reply:
Did not need this step as we are handling banner image in ticket #447